streams/src/test/java/org/apache/kafka/streams/state/internals/TimestampedTimestampedKeyAndJoinSideSerializerTest.java

```diff
@@ -24,49 +24,49 @@
 import static org.hamcrest.Matchers.notNullValue;
 import static org.junit.Assert.assertThrows;
 
-public class KeyAndJoinSideSerializerTest {
+public class TimestampedTimestampedKeyAndJoinSideSerializerTest {
```

Comment:
The class starts with repeated timestamped -> `TimestampedTimestamped...`

Reply:
This comment was not addressed yet.

Reply:
Ack.